data/block/headerWithValidatorStats.go

```diff
@@ -340,6 +340,17 @@ func (hv *HeaderWithValidatorStats) SetShardID(_ uint32) error {
 	return nil
 }
 
+// SetValidatorStatsRootHash sets the root hash for the validator statistics trie
+func (hv *HeaderWithValidatorStats) SetValidatorStatsRootHash(rHash []byte) error {
```

Comment:
rootHash

Reply:
done

Reply:
This is the same everywhere - this is legacy. You would need to change a lot on chain-go repo when integrating such change